schemas/dab.draft.schema.json

```
@@ -207,6 +227,13 @@
           "description": "Global hosting configuration",
           "additionalProperties": false,
           "properties": {
+            "max-response-size-mb": {
+              "type": "integer",
```

```suggestion
             "type": [
                "integer",
                "null"
              ]
```

as this isn't a required field in the config.